docs/allneeds.rst

```diff
@@ -7,10 +7,39 @@ allneeds
 
 Provides an overview of the needs of the fort in general, output is sorted to
 show most unfullfiled needs.
+show only needs that needs attention now (damaging focus) and garantee it shows on dwarf overview
+(obs: multiple Prays counts as unique Pray need)
+
+NEEDS_CODE is usually capital letters from each need
```

Comment:
Could you explain a bit how nicknaming the dwarves makes it easier to care for them? instead of nicknaming dwarves, would it be useful to show which dwarves have which needs in a visual list? Clicking on the dwarf name could jump the screen to that dwarf so the player can take action. The list could be left up as a status panel so the player can refer to it as they are assigning dwarves to things that will satisfy them.

Reply:
in my game im using this mod in this way, i put repeat allneeds u, and have a custom labor CO for craft object for crafts with wood,stone,bone and add the dwarves with CO need and remove when CO is satisfied, 

and another frequent use is martial training, i have 6 squads with no uniform, that i use dwarf therapist to assign into them if nickname contains MT, and after some time, i filter dwarfs with squad that not contains MT in nickname to remove it from squad/training

and less often i have a custom labor for stone encraving, that i assign is nick contains BC be creative

if nick contains PM pray or meditate i can barrow all inside temple to pray

usually have 50 dwarfs with same needs, be able to manage multiple without clicking one by one is the motivation to use this, if i need to access the name again i send allneeds r

---
about your suggestion, if its is possible to select multiples using gui after listing it (but currently i dont know how to code it, but im interested to learn or pair prog this code if its possible), it will be useful as alternative for naming it or one more option, because renaming make compatible to use with dwarf therapist